client/src/localization/languages/ZhTraditional.tsx

```diff
@@ -0,0 +1,256 @@
+  com_endpoint_export: '導出',
+  com_endpoint_save_as_preset: '另存預設值',
+  com_endpoint_presets_clear_warning: '您確定要清除所有預設值嗎？此操作不可逆轉。',
+  com_endpoint_not_implemented: '未實施',
+  com_endpoint_no_presets: '尚未設置預設值',
+  com_endpoint_not_available: '沒有可用的端點',
+  com_endpoint_view_options: '檢視選項',
+  com_endpoint_save_convo_as_preset: '將對話另存為預設值',
+  com_endpoint_my_preset: '我的預設值',
+  com_endpoint_agent_model: '代理模型（建議：GPT-3.5）',
+  com_endpoint_completion_model: '完成模型（建議：GPT-4）',
+  com_endpoint_func_hover: '啟用將插件用作 OpenAI 函數',
+  com_endpoint_skip_hover: '跳過完成步驟，檢查最終答案和生成步驟',
+  com_endpoint_config_key: '設置 API 密鑰',
+  com_endpoint_config_key_for: '為{0}設置API密鑰',
+  com_endpoint_config_key_name: '鍵',
+  com_endpoint_config_value: '輸入{0}的值',
+  com_endpoint_config_key_name_placeholder: '請先設置 API 密鑰',
+  com_endpoint_config_key_encryption: '您的密鑰將被加密並於',
+  com_endpoint_config_key_expiry: '到期時刪除',
+  com_endpoint_config_key_import_json_key: '導入服務帳戶 JSON 鍵。',
+  com_endpoint_config_key_import_json_key_success: '成功導入服務帳戶 JSON 鍵',
+  com_endpoint_config_key_import_json_key_invalid:
+    '無效的服務帳戶 JSON 鍵，您是否導入了正確的文件？',
+  com_endpoint_config_key_get_edge_key: '要獲取 Bing 的訪問權杖，請登錄到',
+  com_endpoint_config_key_get_edge_key_dev_tool:
+    '在訪問該網站時使用開發工具或擴展程序從 _U cookie 中複製其內容。如果此方法失敗，請按照這些',
+  com_endpoint_config_key_edge_instructions: '說明',
+  com_endpoint_config_key_edge_full_key_string: '獲取完整的 Cookie 字符串。',
+  com_endpoint_config_key_chatgpt: '要獲取 ChatGPT \'免費版本\' 的訪問權杖，請登錄到',
+  com_endpoint_config_key_chatgpt_then_visit: '然後訪問',
+  com_endpoint_config_key_chatgpt_copy_token: '複製訪問權杖。',
+  com_endpoint_config_key_google_need_to: '您需要',
+  com_endpoint_config_key_google_vertex_ai: '在 Google Cloud 上啟用 Vertex AI',
+  com_endpoint_config_key_google_vertex_api: 'API，然後',
+  com_endpoint_config_key_google_service_account: '創建一個服務帳戶',
+  com_endpoint_config_key_google_vertex_api_role:
+    '請確保單擊\'創建和繼續\'並至少將\'Vertex AI 使用者\'角色授予。最後，創建一個 JSON 鍵，在此處導入。',
+  com_nav_export_filename: '文件名稱',
+  com_nav_export_filename_placeholder: '設置文件名稱',
+  com_nav_export_type: '類型',
+  com_nav_export_include_endpoint_options: '包含端點選項',
+  com_nav_enabled: '已啟用',
+  com_nav_not_supported: '不支援',
+  com_nav_export_all_message_branches: '導出所有訊息分支',
+  com_nav_export_recursive_or_sequential: '遞歸還是順序？',
+  com_nav_export_recursive: '遞歸',
+  com_nav_export_conversation: '導出對話',
+  com_nav_theme: '主題',
+  com_nav_theme_system: '系統',
+  com_nav_theme_dark: '黑暗',
+  com_nav_theme_light: '亮色',
+  com_nav_clear_all_chats: '清除所有聊天',
+  com_nav_confirm_clear: '確認清除',
+  com_nav_close_sidebar: '關閉側邊欄',
+  com_nav_open_sidebar: '打開側邊欄',
+  com_nav_log_out: '登出',
+  com_nav_user: '用戶',
+  com_nav_clear_conversation: '清除對話',
+  com_nav_clear_conversation_confirm_message:
+    '您確定要清除所有對話嗎？此操作不可逆轉。',
+  com_nav_help_faq: '幫助與常見問題解答',
+  com_nav_settings: '設置',
+  com_nav_search_placeholder: '搜索訊息',
+  com_nav_setting_general: '一般',
+  com_nav_setting_data: '數據控制',
+  com_nav_language: '語言',
```

Comment:
can you remove this com_nav_language translation?

Reply:
done